test/integration/test_integration_basics.py

```diff
@@ -271,3 +271,16 @@ def test_https_log_verbosity(https_test_server_fixture):
       ["--duration 1", "--rps 1", "-v trace",
        https_test_server_fixture.getTestServerRootUri()])
   assertIn(trace_level_sentinel, logs)
+
+
+def test_dotted_output_format(http_test_server_fixture):
+  """
+  Ensure we get the dotted string output format when requested.
+  and ensure we get latency percentiles.
+  """
+  output, _ = http_test_server_fixture.runNighthawkClient([
+      "--duration 1", "--rps 10", "--output-format dotted",
+      http_test_server_fixture.getTestServerRootUri()
+  ],
+                                                          as_json=False)
```

Comment:
This seems very strange formatting, are we running pyformat on the NH repo? See what Envoy does.

Reply:
This is what `tools/check_format fix` outputs. I double checked it, because I also thought it looked rather strange. Maybe we're hitting some weird edge case?
And yup, we are running formatting checks in CI (and they are content with this).

Reply:
Fair enough, kind of strange though.